options/options_test.go

```diff
@@ -4,47 +4,58 @@ import (
 	"testing"
 )
 
-type ListOptions struct {
```

Comment:
You don't really need to prefix those struct with Tests. IMHO it's just confusing

Reply:
Done